### DISCUSSION TITLE
USITT 2024 Attendees

### DISCUSSION THREAD

**Discussion by SparkyBobo** [SparkyBobo/dmxdemonstrator#11] [General]
If you had a chance to attend the Teaching DMX512 in the Networking Age, thanks for attending and listening. If not, sorry we missed each other but we're glad you are here!

If you need any help getting started, feel free to ask questions in this thread, or you can open an issue, or if you want to contribute feel free to submit a pull request!

As of the public announcement at USITT, the documentation is a bit behind the current states but we plan to have this fixed by the time you get home.

If you'd like to build a DMX Demonstrator, please leave a comment, we want to make sure you are successful! If you have any issues or feedback, please reach out so we can get you in a working state. Thanks!

**Comment by akulhavy** [comment]
Hi @SparkyBobo! My name is Andy, you met my wife Maria at USITT - she works with Barbizon NY. We're working together on the DMX demonstrator and had a few questions. Are you okay with me adding pull requests? Would you rather we drop questions/suggestions in this discussion thread, or open issues first? We've only just ordered parts, so most of our questions are pretty surface level but we've still got a few notes. Let us know how you'd like us to share out feedback and thoughts!

**Reply by SparkyBobo** [reply]
Hi! Whatever works for you works for me, pull requests are a fine place to have a discussion in my opinion, whatever format that you think works best!

**Reply by SparkyBobo** [reply]
@akulhavy - I forgot to mention that I recently checked in some BOM and parts files for most of the boards (the rest are staged on my computer), this might help answer some questions. Checkout https://github.com/SparkyBobo/dmxdemonstrator/pull/22
